lib/services/local/configuration.go

```diff
@@ -239,11 +265,11 @@ func (s *ClusterConfigurationService) DeleteClusterConfig() error {
 
 // SetClusterConfig sets services.ClusterConfig on the backend.
 func (s *ClusterConfigurationService) SetClusterConfig(c services.ClusterConfig) error {
-	if err := s.syncClusterConfigWithNetworkingConfig(c); err != nil {
-		return trace.Wrap(err)
+	if c.HasNetworkingFields() {
+		return trace.BadParameter("call SetClusterNetworkingConfig instead")
```

Comment:
Error message is a bit unclear
```suggestion
	if c.HasNetworkingFields() {
		return trace.BadParameter("cluster config has legacy networking fields, call SetClusterNetworkingConfig to set these fields")
```